doc/source/reference/arrays.ndarray.rst

```diff
@@ -158,10 +158,11 @@ base offset itself is a multiple of `self.itemsize`. Understanding
 
 .. note::
 
-    Points (1) and (2) are not yet applied by default. Beginning with
-    NumPy 1.8.0, they are applied consistently only if the environment
-    variable ``NPY_RELAXED_STRIDES_CHECKING=1`` was defined when NumPy
-    was built. Eventually this will become the default.
+    Points (1) and (2) can currently be disabled by the compile time
+    environmental variable ``NPY_RELAXED_STRIDES_CHECKING=0``.
```

Comment:
```suggestion
    environmental variable ``NPY_RELAXED_STRIDES_CHECKING=0``,
    which was the default before NumPy 1.10.
```

Reply:
How far back do we want to actually do such bookkeeping? 1.10 is approaching 5 years ago.